From 57cf819efa50a6a3b3a478c25a098c29722eb358 Mon Sep 17 00:00:00 2001
From: Andy Fiddaman <omnios@citrus-it.co.uk>
Date: Mon, 4 Apr 2022 13:46:52 +0000
Subject: [PATCH] 14617 svc:/network/ipmp maintenance in shared IP zone
 Reviewed by: Marco van Wieringen <mvw@planets.elm.net> Reviewed by: Yuri
 Pankov <ypankov@tintri.com> Reviewed by: Garrett D'Amore <garrett@damore.org>
 Approved by: Robert Mustacchi <rm@fingolfin.org>

---
 usr/src/cmd/cmd-inet/usr.lib/in.mpathd/net-ipmp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/cmd/cmd-inet/usr.lib/in.mpathd/net-ipmp b/usr/src/cmd/cmd-inet/usr.lib/in.mpathd/net-ipmp
index 565b80e374..4dbb84a5b8 100644
--- a/usr/src/cmd/cmd-inet/usr.lib/in.mpathd/net-ipmp
+++ b/usr/src/cmd/cmd-inet/usr.lib/in.mpathd/net-ipmp
@@ -22,7 +22,7 @@ if [ -z "$SMF_FMRI" ]; then
         exit $SMF_EXIT_ERR_NOSMF
 fi
 
-smf_configure_ip || exit $SMF_EXIT_ERR_FATAL
+smf_configure_ip || exit $SMF_EXIT_NODAEMON
 
 if /lib/inet/in.mpathd; then
         exit $SMF_EXIT_OK
-- 
2.42.0

